src/elements/select/select.visual.spec.ts

```diff
@@ -0,0 +1,251 @@
+import { html, nothing, type TemplateResult } from 'lit';
+
+import { describeViewports, visualDiffDefault, visualDiffFocus } from '../core/testing/private.js';
+
+import '../form-error.js';
+import '../form-field.js';
+import '../option.js';
+import './select.js';
+
+describe('sbb-select', () => {
+  const valueEllipsis: string = 'This label name is so long that it needs ellipsis to fit.';
+  const defaultArgs = {
+    borderless: false,
+    negative: false,
+    floatingLabel: false,
```

Comment:
I see no tests with floatingLabel true, should we add or remove floatingLabels from arguments?

Reply:
There are no stories in storybook too; i propose to remove it, since the floating case is handled in the form-field.
What do you think?

Reply:
ok!